bin/rollem.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs')
 const join = require('path').join
 const argv = require('minimist')(process.argv.slice(2))
 
-const configFilename = argv.c || 'rollem.config.js'
+const configFilename = (argv.c === true) ? 'rollem.config.js' : argv.c;
```

Comment:
Could you clarify why inside the Rollem you are not using semicolons? I used to have semicolons in each line of my code, but probably I missed some JS new rule :)